Sources/NostrSDK/Events/Tags/IdentifierTagInterpreting.swift

```diff
@@ -14,4 +14,16 @@ public extension IdentifierTagInterpreting {
     var identifier: String? {
         firstValueForTagName(.identifier)
     }
+
+    /// The event coordinates that can be used to fetch this replaceable event's kind, pubkey, and identifier from a relay.
+    ///
+    /// - Parameters:
+    ///   - relayURL: A relay URL that this replaceable event could be found.
+    func identifierEventCoordinates(_ relayURL: URL? = nil) -> EventCoordinates? {
```

Comment:
I believe every event with a `d` tag should be addressable with event coordinates.